### ISSUE TITLE
Accessibility] Narrator cannot focus on a ListView for the second time(1 listView on form) or the second listView control( 2 listViews on form)

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4030]
* .NET Core Version: 
.Net SDK 5.0.100-rc.2.20477.2

* Have you experienced this same bug with .NET Framework?: 
No

* More info
This is a regression issue, cannot repro in .Net Core 3.1

**Minimal repro:**
1.	Open the attached WinFormsApp9.zip application in VS, which contains 2 ListViews on form, then build and run it.
[WinFormsApp9.zip](https://github.com/dotnet/winforms/files/5302869/WinFormsApp9.zip)

2.	Launch Narrator to focus on ListViews controls

**Actual behavior:**
Narrator cannot focus on the second ListView control by pressing Tab keyboard, the blue rectangle still focus on the first listView control
![image](https://user-images.githubusercontent.com/26474449/94637692-cdaaa100-030a-11eb-9f99-941cee6dccf5.png)

**Expected behavior:**
Narrator cannot focus on every ListView control by pressing Tab keyboard

**Comment by Jenna-Zheng** [comment]
Verified this issue with Dev16 16.9.0 Preview 2.0 [30708.24.main] + .NET 6.0.100-alpha.1.20557.2, this issue has been fixed, please check the follow screenshots:
![image](https://user-images.githubusercontent.com/64949616/98529633-d05cc680-22b8-11eb-8f27-a4310963fdd7.png)
![TestResult_focus](https://user-images.githubusercontent.com/64949616/98529739-f5e9d000-22b8-11eb-9578-d3a20a455447.gif)
